### DISCUSSION TITLE
Talk broken "Waiting for others to join the call" on all devices.

### DISCUSSION THREAD

**Discussion by sol8712** [nextcloud/all-in-one#3148] [Questions] [answered]
### Steps to reproduce
1. Start a call in public talk room.
2. Get message "Waiting for others to join the call …"
3. All joining the call get this message.

### Expected behavior 

Joining a call works

### Actual behavior 

Sits on "Waiting for others to join the call" even when others are connected.

### Host OS 
Ubuntu 22.04 Server

#### Nextcloud AIO version 
6.4.0 did not work 7.0.0 beta also does not work

#### Current channel 
Beta now(stable had the same issue)

#### Other valuable info 
No errors in any logs, it happens on all different devices and browsers, apache2 reverse proxy setup. Text chat works...fine(just no notifications at all)

Back in April or May Talk worked fine, i didnt use the feature for a few months now its broken. I desperately need this working if anyone knows the way or a similar alternative to talk if there's no foreseeable fix

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by sol8712** [comment, marked as answer]
Thank you for the info and all the work you do here!

However after 3 days of frustration its solved for me. The source of my issue with talk (as well as a Collabora issue) was that i set up the aio as a sub-directory in my reverse proxy(example.com/cloud), i have jumped though all the hoops to move my aio install to cloud.example.com and all of my problems disappeared, everything works great!

For others with this situation you must first do this: https://github.com/nextcloud/all-in-one#how-to-change-the-domain then you must also make sure the setting overwritewebroot in the aio-nextcloud config.php is set back to /

Use this to edit the config.php:
`sudo docker run -it --rm --volume nextcloud_aio_nextcloud:/var/www/html alpine sh -c "apk add --no-cache nano && nano /var/www/html/config/config.php"`
